exercises/go-counting/canonical-data.json

```diff
@@ -0,0 +1,172 @@
+{
+  "exercise": "go-counting",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Test for black territory on 5x5 board",
```

Comment:
I suggest to remove the words `test for` or `test to` from the descriptions because it is already the case that everything in this file is a test; the descriptio ndoes not need to reaffirm it. Like in https://github.com/exercism/problem-specifications/pull/771.